src/button/createbutton.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import { getEnvKeystrokeText } from '../../utils/keyboard.js';
+
+/**
+ * Creates an instance of {@link ui.button.ButtonView} class using
+ * defined model.
+ *
+ * @param {ui.button.ButtonModel} model Model of this button.
+ * @param {utils.Locale} locale The {@link core.editor.Editor#locale editor's locale} instance.
+ * @returns {ui.button.ButtonView} The button view class constructor.
+ */
+export default function createButton( model, locale, ButtonView ) {
+	const buttonView = new ButtonView( locale );
+
+	buttonView.bind( 'label', 'isOn', 'isEnabled', 'withText', 'type', 'icon' ).to( model );
+	buttonView.bind( 'title' ).to( model, 'label', model, 'keystroke', ( label, keystroke ) => {
+		if ( keystroke ) {
+			label += ` (${ getEnvKeystrokeText( keystroke ) })`;
+		}
+
+		return label;
+	} );
```

Comment:
I remember we agreed that the `title` should be built in the `ButtonView`. It should have `title` property, which, if set, is used. Otherwise, a label with a keystroke (if exists) is used.